commands.js

```diff
@@ -603,7 +603,7 @@ exports.commands = {
 	roomdesc: function (target, room, user) {
 		if (!target) {
 			if (!this.runBroadcast()) return;
-			if (!room.desc) return this.sendReply("This room does not have a description set.");
+			if (!room.desc) return this.errorReply("This room does not have a description set.");
```

Comment:
https://github.com/Zarel/Pokemon-Showdown/blob/master/commands.js#L676

That was before I made the PR, so I just realized that was an error too.